src/output.cpp

```diff
@@ -2011,99 +2012,106 @@ static void do_kw_subst(chunk_t *pc)
 } // do_kw_subst
 
 
-static void output_comment_multi_simple(chunk_t *pc, bool kw_subst)
+static void output_comment_multi_simple(chunk_t *pc)
 {
-   UNUSED(kw_subst);
+   if (pc == nullptr)
+   {
+      return;
+   }
+
    cmt_reflow cmt;
    output_cmt_start(cmt, pc);
 
-   int col_diff = 0;
-   if (chunk_is_newline(chunk_get_prev(pc)))
-   {
-      // The comment should be indented correctly
-      col_diff = pc->orig_col - pc->column;
-   }
-   else
-   {
-      // The comment starts after something else
-      col_diff = 0;
-   }
+   // The multiline comment is saved inside one chunk. If the comment is
+   // shifted all lines of the comment need to be shifter by the same amount.
+   // Save the difference of initial and current position to apply it on every
+   // line_column
+   const int col_diff = [pc](){
```

Comment:
Is the lambda really required here?

Reply:
Well not really but I wanted the `col_diff` to be const.